docs/using-gitbase/examples.md

```diff
@@ -55,3 +55,15 @@ FROM (
 ) as t
 GROUP BY committer_email, month, repo_id
 ```
+
+## Create an index for columns on a table
+
```

Comment:
You can also add a paragraph explaining that you can create an index using more than one column at a time.